src/js/tabs/landing.js

```diff
@@ -2,18 +2,51 @@
 
 TABS.landing = {};
 TABS.landing.initialize = function (callback) {
-    var self = this;
+  var self = this;
```

Comment:
This file will need to have the indentation adjusted at some point (not in this pull request though).